src/Symfony/Component/Console/Tests/Completion/CompletionInputTest.php

```diff
@@ -133,4 +133,19 @@ public static function provideFromStringData()
         yield ['bin/console cache:clear "multi word string"', ['bin/console', 'cache:clear', '"multi word string"']];
         yield ['bin/console cache:clear \'multi word string\'', ['bin/console', 'cache:clear', '\'multi word string\'']];
     }
+
+    public function testToString()
```

Comment:
We could perhaps use a dataProvider for this test to avoid duplicating code ?

Reply:
Before adding this, I read `ArgvInputTest` and `ArrayInputTest`. There both don't use a dataProvider. Personally, I think use or not use `dataProvider` here is fine because there are only 2-3 cases.

If it's neccessary, I will implement. And we should implement for `ArgvInputTest` and `ArrayInputTest`, shouldn't we?

Reply:
Oh no, it's not necessary, it's really just a matter of separating the test data from the test itself, but you're right, the code is simple and it's all detail.